addon/components/line-chart/component.js

```diff
@@ -0,0 +1,253 @@
+import moment from 'moment';
+import _ from 'lodash/lodash';
+import d3 from 'd3';
+import dc from 'dc';
+import crossfilter from 'crossfilter';
+// import $ from 'jquery';
+import BaseChartComponent from '../base-chart-component';
+
+/**
+   @public
+   @module line-chart
+   @type component
+   @desc dc.js line chart
+*/
+export default BaseChartComponent.extend({
+    classNames: ['line-chart'],
+
+    colors: [
```

Comment:
We should pull colors out of our chart components and share them.

Reply:
Will do.